pkg/containerutil/containerutil.go

```diff
@@ -413,3 +418,150 @@ func waitContainerStop(ctx context.Context, exitCh <-chan containerd.ExitStatus,
 		return status.Error()
 	}
 }
+
+// Cp implements `nerdctl cp`.
+//
+// See https://docs.docker.com/engine/reference/commandline/cp/ for the specification.
+func Cp(ctx context.Context, container2host bool, pid int, dst, src string, followSymlink bool) error {
```

Comment:
Should this move to a separate file for linux (e.g., `cp_linux.go`) since it's from `container_cp_linux.go`.

Reply:
Make sense, I'll move it.

Reply:
I have also renamed `pkg/cmd/container/cp.go` to `.../cp_linux.go` because the _cp command entry_ is derived from `container_cp_linux.go` too. Seems the `cmd/nerdctl/container_cp.go` is just file with util functions, is it？